WordPress/Classes/ViewRelated/Blog/SharingViewController.m

```diff
@@ -103,6 +135,13 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     return cell;
 }
 
+- (void)cellButtonTapped:(UIButton *)sender
+{
+    UITableViewCell *tappedCell = (UITableViewCell *)[sender superview];
```

Comment:
(Just in case this changes at some point!) maybe an assert of tappedCell's kind?